requirements/edx/base.in

```diff
@@ -47,7 +47,7 @@ django-method-override==0.1.0
 django-model-utils==3.0.0
 django-mptt>=0.8.6,<0.9
 django-mysql
-django-oauth-toolkit==0.12.0
+django-oauth-toolkit<1.2            # 1.2+ requires Django 2 and Python 3.5
```

Comment:
@jmbowman: Does it seem reasonable to add comments like this?  Most of our comments are about what the dependency is, and not around what is holding up an upgrade.

Reply:
Sure, this is fine.  Even better is to have both a note regarding what we use it for and the reason for restricting it

Reply:
Updated.